src/test/resources/com/puppycrawl/tools/checkstyle/checks/modifier/interfacememberimpliedmodifier/InputInterfaceMemberImpliedModifierNestedOnInterface.java

```diff
@@ -0,0 +1,60 @@
+package com.puppycrawl.tools.checkstyle.checks.modifier.interfacememberimpliedmodifier;
+
+public interface InputInterfaceMemberImpliedModifierNestedOnInterface {
+
+    public static interface NestedInterfacePublicStatic {
+    }
+
+    public interface NestedInterfacePublic {
+    }
+
+    static interface NestedInterfaceStatic {
+    }
+
+    interface NestedInterface {
+    }
+
+    public static enum NestedEnumPublicStatic {
+        TRUE,
+        FALSE
+    }
+
+    public enum NestedEnumPublic {
+        TRUE,
+        FALSE
+    }
+
+    static enum NestedEnumStatic {
+        TRUE,
+        FALSE
+    }
+
+    enum NestedEnum {
+        TRUE,
+        FALSE
+    }
+
+    public static class NestedClassPublicStatic {
+    }
+
+    public class NestedClassPublic {
+    }
+
+    static class NestedClassStatic {
+    }
+
+    class NestedClass {
+    }
+
+    public default boolean methodWithLocalClass(String input) {
+        class LocalClass {
```

Comment:
from https://github.com/checkstyle/checkstyle/pull/5974#pullrequestreview-132677925

Passing CI is good, but not enough, we need prove that there are no exceptions on real life ugly projects.
Please read README of https://github.com/checkstyle/contribution/tree/master/checkstyle-tester
Please launch launch.groovy on all projects that are referenced in property file(just uncomment all). There should be no exceptions. Unfortunately there will be too much violations to share in web, so please select 1-3 projects you like(openjdkX is good candidate) and run report generation on them only and share it on github.io for us to review.
If you have any concerns or questions do not hesitate to ask us.